### PR TITLE
vote model with testing

### PR DESCRIPTION
**Pull request by storycoder**
@timcheadle how does this look so far (tests)? I know I need to configure the model.

### CODE REVIEW

File: test/models/movie_test.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    # @movie = Movie.new
+    # @movie.title = 'Big Lebowski'
+    # assert @movie.valid?
+
+    # @movie.title = nil
+    # refute @movie.valid?
+
+    # @movie.title = ''
+  #   # refute @movie.valid?
+  # end
```
Comment (timcheadle) [inline test/models/movie_test.rb]: You can just delete these comments if you want.

File: test/models/movie_test.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  # 	@movie = Movie.new
+  # 	@movie.url = "http://www.biglebowski.com"
+  # 	assert @movie.valid?
+
+  # 	@movie.url = nil
+  # 	refute @movie.valid?
+
+  # 	@movie.url = ''
+  # 	refute @movie.valid?
+  # end
```
Comment (timcheadle) [inline test/models/movie_test.rb]: You can delete these comments too.

File: test/models/vote_test.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+	test "requires an event" do 
+		@vote.event = nil
+		refute @vote.valid?
+	end
+
+	test "requires a movie" do
+		@vote.movie = nil
+		refute @vote.valid?
+	end
```
Comment (timcheadle) [inline test/models/vote_test.rb]: The shoulda matchers below make these tests above redundant.

**Comment by timcheadle** [comment]
Everything mostly looks good! There are some issues with how you've done the associations, but we'll cover that in class next time. Feel free to make changes I mentioned above and merge this.

**Comment by storycoder** [comment]
@timcheadle thanks for getting back to me. Before merging, I made some final changes to my app. Please my last commit. Did I improve my associations on the vote model?

**Pull request merged**